docs/documentation/build.md

```diff
@@ -144,6 +144,16 @@ Flag                | `--dev` | `--prod`
 </details>
 
 <details>
+  <summary>inline</summary>
+  <p>
+    `--inline` _default: true_
```

Comment:
I share @clydin's concerns on this: there is a benefit and a demand for this functionality, but we neither should change the existing behaviour nor put the extra responsibility of extra CSP configuration on users. 

It might be trivial for some users, but not so much for others, and possibly completely impractical for some server setups.

For this reason I think it should default to false.